engines/order_management/app/services/order_management/order/updater.rb

```diff
@@ -57,11 +57,12 @@ def update_item_total
       end
 
       def update_adjustment_total
-        order.adjustment_total = adjustments.eligible.sum(:amount)
+        order.adjustment_total = adjustments.eligible.sum(:amount) +
+                                 all_adjustments.shipping.sum(:amount)
```

Comment:
Does this mean that shipping adjustments are not eligible? I was just wondering why we don't just do `all_adjustments.eligible.sum(:amount)`.

Reply:
The shipping adjustment isn't included in the `order.adjustments` scope now. Shipping adjustments are eligible. 

The `eligible` scope is an interesting one, in Spree 2.2 they actually remove it completely. I tried removing it previously in a quick test and it turns out it's still used in OFN, but only in relation to payment adjustments. So all adjustments are basically eligible expect for payment fees, where some are and some are not.

Reply:
`all_adjustments.eligible.sum(:amount)` would return the wrong amount in this case, but... maybe we can do `all_adjustments.eligible.additional.sum(:amount)` :+1:

Reply:
Wow, what a complex beast it still is.